python/lsst/ci/models.py

```diff
@@ -0,0 +1,138 @@
+from dataclasses import dataclass
+from typing import List, Set, Optional, OrderedDict as ODict
+from collections import OrderedDict
```

Comment:
Please don't use OrderedDict in new code (and, as suggested earlier) please also remove its use in existing code if you are willing.